homeassistant/components/elmax/alarm_control_panel.py

```diff
@@ -93,12 +95,19 @@ async def async_alarm_disarm(self, code: str | None = None) -> None:
         )
         await self.coordinator.async_refresh()
 
+    @property
+    def state(self) -> StateType:
+        """Return the state of the entity."""
+        return (
+            None
+            if self._last_state is None
+            else ALARM_STATE_TO_HA.get(self._last_state.armed_status)
+        )
```

Comment:
```suggestion
        if (state := self._last_state) is not None:
            return ALARM_STATE_TO_HA.get(state.armed_status)
        return None
```
Please avoid inline if statements if they take more than 1 line